migrations/versions/8454ee309fcc_initial_migration.py

```diff
@@ -0,0 +1,44 @@
+"""initial migration
+
+Revision ID: 8454ee309fcc
```

Comment:
interesting how it uses revision id to name the migration files

Reply:
ActiveRecord works similarly, labeling each migration with a unique ID, except it uses the timestamp of when the migration was created.